api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/PaymentCycleEmailHandler.java

```diff
@@ -46,6 +70,76 @@ private void sendChildTurnsFourEmail(PaymentCycle paymentCycle, NextPaymentCycle
         messageQueueClient.sendMessage(messagePayload, MessageType.SEND_EMAIL);
     }
 
+    private EmailMessagePayload buildChildTurnsFourNotificationEmailPayload(PaymentCycle paymentCycle,
+                                                                            PaymentCycleVoucherEntitlement entitlementNextMonth,
+                                                                            boolean multipleChildrenTurningFourInMonth) {
+        Map<String, Object> emailPersonalisation = createPaymentEmailPersonalisationMap(paymentCycle, entitlementNextMonth);
+        emailPersonalisation.put(MULTIPLE_CHILDREN.getTemplateKeyName(), multipleChildrenTurningFourInMonth);
+        return EmailMessagePayload.builder()
+                .claimId(paymentCycle.getClaim().getId())
+                .emailType(EmailType.CHILD_TURNS_FOUR)
+                .emailPersonalisation(emailPersonalisation)
+                .build();
+    }
+
+    private Map<String, Object> createPaymentEmailPersonalisationMap(PaymentCycle paymentCycle, PaymentCycleVoucherEntitlement voucherEntitlement) {
```

Comment:
suggest we name this something like `createEmailPersonalisationMapForNextCycle`

Reply:
Renamed.